File: demo/springboot/demo/src/main/java/com/demo/domain/dto/http/resonse/OssBatchUploadResponse.java
package com.demo.domain.dto.http.resonse;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.minio.domain.bo.UploadItemResult;
import lombok.Data;

import java.util.List;

@Data
public class OssBatchUploadResponse {
    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private Long userId;
    private String bucketName;
    private Integer successCount;
    private Integer failCount;
    private List<UploadItemResult> items;
}

File: demo/kmp/shared/src/commonMain/kotlin/com/vectordemo/domain/dto/http/response/OssBatchUploadResponse.kt
package com.vectordemo.domain.dto.http.response

import kotlinx.serialization.Serializable

@Serializable
data class OssBatchUploadResponse(
    val userId: String? = null,
    val bucketName: String? = null,
    val successCount: Int? = null,
    val failCount: Int? = null,
    val items: List<OssUploadItemResult>? = null,
)
